src/scripts/quests/QuestLineHelper.ts

```diff
@@ -237,7 +237,7 @@ class QuestLineHelper {
     }
 
     public static createRocketjohtoQuestLine() {
-        const RocketjohtoQuestLine = new QuestLine('Radio Tower Takeover', 'Team Rocket is up to no good again!');
+        const RocketjohtoQuestLine = new QuestLine('Team Rocket', 'Team Rocket is up to no good again!');
```

Comment:
Aw I liked the old name, although it kinda sounds like you're taking over the radio tower 😅

Reply:
Name change because the questline doesn't just involve the tower anymore.